src/Containers/Microsoft.NET.Build.Containers/ContainerBuilder.cs

```diff
@@ -134,7 +145,7 @@ public static class ContainerBuilder
         return 0;
     }
 
-    private static LocalDocker GetLocalDaemon(string localDaemonType, Action<string> logger)
+    private static LocalDocker GetLocalDaemon(string localDaemonType, Action<LogMessage> logger)
```

Comment:
minor: `ILogger` makes this signature more c-sharpie

Reply:
Changed in favor of `ILogger` implementation